sound/sequences/00_sound_player.s

```diff
@@ -6986,6 +6986,42 @@ sound_ref .sound_menu_power_meter
 sound_ref .sound_menu_camera_buzz
 sound_ref .sound_menu_camera_turn
 
+// Crash screen error sfx
+
+sound_ref .sound_menu_critical_error
+
+.envelope_critical_error:
+envelope_line 1 32760
+envelope_line 128 0
+envelope_line 500 500
+envelope_goto 4
```

Comment:
This envelope is extremely scuffed. The `500 500` line is taking the sound from 0 and actually adding to it first off (so just remove that line). Secondly, that `goto` line is very bad news; please use a hang for that instead.

Also this should be moved to the bottom of the file, where envelopes go.